src/static/css/pages/beta.less

```diff
@@ -161,10 +157,12 @@
 .progress-charts_chart {
     width: 100px;
     height: 100px;
+    margin-right: 1em;
+    float: left;
 
-    .respond-to-max(479px, {
-        margin-right: 1em;
-        float: left;
+    .respond-to-min(@tablet-min, {
```

Comment:
Rather than reset to the default values and double the code, we should keep the max mixin with `@mobile-max`.